src/foam/u2/grid/GridCell.js

```diff
@@ -7,6 +7,10 @@ foam.CLASS
     imports: [
     ],
     
+    requires: [
+        'com.serviceecho.dao.ContextualizingSink', 
```

Comment:
foam code can't require serviceecho code. This would need to be moved to foam as well. Or make everything into a com.serviceecho.* open-source package.